app/models/policy.rb

```diff
@@ -1,19 +1,27 @@
 class Policy < ApplicationRecord
   belongs_to :plan
+  belongs_to :user
   has_one :bank
   has_one :medical_history
-  has_one :vehicle
+  has_one :vehicle   
```

Comment:
@VaibhaviJoshi-BTC : Spacing issue after vehicle
Write acceept nested attributes for the required association after its declaration
like has_one :vehicle 
accepts_nested_attributes_for :vehicles